docs-web/app-protect/troubleshooting.md

```diff
@@ -102,4 +106,21 @@ This timeout should be more than enough to verify configurations. However, when
 - You need to apply a large amount of Ingress Resources at once.
 - You are running the Ingress Controller for the first time in a cluster where the Ingress Resources with App Protect enabled are already present.
 
-You can increase this timeout by setting the `nginx-reload-timeout` [cli-argument](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#cmdoption-nginx-reload-timeout).
\ No newline at end of file
+You can increase this timeout by setting the `nginx-reload-timeout` [cli-argument](/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#cmdoption-nginx-reload-timeout).
+
+If you are using external references in your Nginx App Protect policies, verify if the servers hosting the referenced resources are available and that their response time is as short as possible (see next paragraph).
+If the references are not available during ingress controller startup the pod will fail to startup. In case the resources are not available during a reload, the reload will fail, and ingress controller will process traffic with the previous correct configuration.
+
+### Check the HTTP response time to resources used in App Protect policy external references.
```

Comment:
This should be moved under the Troubleshooting Methods section, as it is a troubleshooting method.

I suggest two small enhancements:
* For brevity, renaming it to "Check the Availability of APPolicy External References" 
* Adding the disclaimer like this:
This method only applies if you're using [external references](https://docs.nginx.com/nginx-app-protect/configuration/#external-references) in NGINX App Protect policies.